lib/Translation/Translator.php

```diff
@@ -447,4 +459,15 @@ public function __call($method, $args)
     {
         return call_user_func_array([$this->translator, $method], $args);
     }
+
+    /**
+     * @param string $domain
+     * @param string $locale
+     *
+     * @return string
+     */
+    private function getCacheKey($domain, $locale)
```

Comment:
Why not real types instead?

```suggestion
    private function getCacheKey(string $domain, string $locale): string
```